state/store.go

```diff
@@ -37,6 +39,18 @@ func calcABCIResponsesKey(height int64) []byte {
 	return []byte(fmt.Sprintf("abciResponsesKey:%v", height))
 }
 
+var tmpABCIKey []byte
+
+func init() {
+	var err error
+	// temporary extra key before consensus param protos are regenerated
+	// TODO(wbanfield) remove in next PR
+	tmpABCIKey, err = orderedcode.Append(nil, int64(10000))
```

Comment:
Assuming this is deleted in a future PR, this is fine. `orderedcode` is not used in `main` but was in `v0.36.x` so reference to it should not outlive this set of pull requests.

Reply:
Yes :-)
In the next one